File: SDKs/amazonaws.com/clouddirectory/2016-05-10/java/lib/src/main/java/openapisdk/models/shared/LimitExceededException.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * LimitExceededException
 * Indicates that limits are exceeded. See <a href="http://docs.aws.amazon.com/directoryservice/latest/admin-guide/limits.html">Limits</a> for more information.
**/
public class LimitExceededException {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("Message")
    public String message;
    public LimitExceededException withMessage(String message) {
        this.message = message;
        return this;
    }
}

File: SDKs/amazonaws.com/clouddirectory/2016-05-10/python/src/sdk/models/shared/limitexceededexception.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class LimitExceededException:
    r"""LimitExceededException
    Indicates that limits are exceeded. See <a href=\"http://docs.aws.amazon.com/directoryservice/latest/admin-guide/limits.html\">Limits</a> for more information.
    """
    
    message: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Message') }})
    
